rca/project_styleguide/templates/patterns/atoms/pathway-item/pathway-item.html

```diff
@@ -1,6 +1,6 @@
 <div class="pathway-item">
     <h4 class="pathway-item__heading body body--one">
-        <div class="pathway-item__heading-container">{{ path.title }}</div>
+        <div class="pathway-item__heading-container">{{ path.heading }}</div>
```

Comment:
Shouldn't we check if heading is empty first before displaying the markup? Same for preview_text, they both seem to be not required.